pkg/controller/datavolume/controller-base.go

```diff
@@ -761,7 +746,9 @@ func (r *ReconcilerBase) newPersistentVolumeClaim(dataVolume *cdiv1.DataVolume,
 	if util.ResolveVolumeMode(targetPvcSpec.VolumeMode) == corev1.PersistentVolumeFilesystem {
 		labels[common.KubePersistentVolumeFillingUpSuppressLabelKey] = common.KubePersistentVolumeFillingUpSuppressLabelValue
 	}
-	labels = passDataVolumeInstancetypeLabelstoPVC(dataVolume.GetLabels(), labels)
+	for k, v := range dataVolume.Labels {
```

Comment:
Note that not all flows we have are using `newPersistentVolumeClaim`
One example I could find is smart clone which creates the target PVC in
https://github.com/kubevirt/containerized-data-importer/blob/b875775732019b3f1f3c614c763bfca508c06bb5/pkg/controller/datavolume/smart-clone-controller.go#L370-L374

Reply:
Good point!

Reply:
Hi Alex, there is no DV in that function and the PVC is created from a VolumeSnapshot, so I don't think labels should be passed in this case. wdyt ?